cmd/ccboc/cmd/commands.go

```diff
@@ -0,0 +1,278 @@
+package cmd
+
+import (
+	"bytes"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"os/user"
+	"path/filepath"
+	"strings"
+
+	"github.com/jedib0t/go-pretty/v6/table"
```

Comment:
Do we still need this library? Does cobra have its own output style?

Reply:
Didnt come across such information.